src/number_theory/number_field.lean

```diff
@@ -49,16 +49,10 @@ lemma int.not_is_field : ¬ is_field ℤ :=
 
 namespace number_field
 
-variables (K L : Type*) [field K] [field L] [nf : number_field K]
+variables (K L : Type*) [field L]
```

Comment:
I feel like this is almost a type synonym - if we want the integral closure that can be done the other way. I don't think anyone ever talks about `O R` for `R` a non-field